src/main/resources/CookieConsent/WebHome.xml

```diff
@@ -71,11 +72,15 @@
       <activeCookieTypes>
         <cache>0</cache>
         <customDisplay/>
+        <defaultValue/>
         <disabled>0</disabled>
         <displayType>checkbox</displayType>
+        <freeText/>
+        <hint/>
+        <largeStorage>0</largeStorage>
```

Comment:
These are not supported by the XWiki 8.x which is used as parent currently.

Reply:
Instead of using 13.10.11, on which I changed related files of extension, must be used version 8.4-12?

Reply:
You need to export from XWiki 8.4. New XAR format is usually backwards compatible which means old XARs continue to work on new XWiki version, but the other way around is not true: installing pages with the new XAR format in old XWiki instances is not guaranteed to work.

Reply:
I will wait for updated parent up to 11.10. And then export from XWiki 11.10.